src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -785,4 +574,217 @@ public static Image getGameIcon(final Window frame) {
     }
     return img;
   }
+
+  public static void startNewTripleA(final Long maxMemory) {
+    startGame(System.getProperty(GameRunner.TRIPLEA_GAME_PROPERTY), null, maxMemory);
+  }
+
+  public static void startGame(final String savegamePath, final String classpath, final Long maxMemory) {
+    final List<String> commands = new ArrayList<>();
+    if (maxMemory != null && maxMemory > (32 * 1024 * 1024)) {
+      ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath, maxMemory);
+    } else {
+      ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath);
+    }
+    if (savegamePath != null && savegamePath.length() > 0) {
+      commands.add("-D" + GameRunner.TRIPLEA_GAME_PROPERTY + "=" + savegamePath);
+    }
+    // add in any existing command line items
+    for (final String property : GameRunner.COMMAND_LINE_ARGS) {
```

Comment:
That's what I mean... We are iterating through the Array instead of using a Collection